### DISCUSSION TITLE
[feature request] show uncited footnotes

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#30891] [Other Feature Feedback, Questions, & Ideas]
In GitHub's Markdown, footnotes are displayed as

```markdown
Here is a simple footnote[^1].

[^1]: My reference.
```

![screenshot](https://user-images.githubusercontent.com/181628/186848204-d85831cd-566c-4fb8-9d7e-cbaaa5d201e1.png)

If a footnote is present, but not referenced, nothing is shown:

```markdown
Here is a simple footnote.

[^1]: My reference.
```

![screenshot](https://user-images.githubusercontent.com/181628/186848351-3987e578-1c39-41b2-8637-bbfef5fbcb4e.png)

It'd be great if there was a way for showing that footnote nonetheless, perhaps even by default.

**Comment by radar-blip** [comment]
I dont know. I think they need to be linked one way or another (thats why the numbers).
Might work for only a few sentences.
